apps/chat/agent/tools.py

```diff
@@ -26,19 +26,45 @@
     from apps.pipelines.models import Node
 
 
+OCS_CITATION_PATTERN = r"<CIT\s+file_id=(?P<file_id>\d+)\s*/>"
+
 SUCCESSFUL_ATTACHMENT_MESSAGE: str = "File {file_id} ({name}) is attached to your response"
 
 CREATE_LINK_TEXT = """You can use this markdown link to reference it in your response:
     `[{name}](file:{team_slug}:{session_id}:{file_id})` or `![](file:{team_slug}:{session_id}:{file_id})`
     if it is an image.
 """
 
-CHUNK_RESULT_TEMPLATE = """
-# File: {file_name}
-## Content
+CHUNK_TEMPLATE = """
+## Chunk: {chunk_id}
+### File name: {file_name}, file_id={file_id}
+### Content
 {chunk}
 """
 
+CHUNK_RESULT_TEMPLATE = """
+# Retrieved chunks
+{retrieved_chunks}
+
+**CRITICAL REQUIREMENT - MANDATORY CITATIONS:**
+
+You MUST cite all information using this exact format: <CIT file_id=the-file-id />
```

Comment:
what about just `<CIT the-file-id>`? If you want it to look like XML then maybe add quotes: `<CIT file_id="the-file-id"/>`. Either way you should probably handle quotes and no quotes.